### PR TITLE
Peakidentify cleaning in progress and supervised ML in progress

### PR DESCRIPTION
**Pull request by kernb2**

### CODE REVIEW

File: ramandecompy/peakidentify.py [hunk 1]
```diff
@@ -612,6 +610,7 @@ def process_score(unknown_peaks,known_peaks,k, precision, unknownname, knownname
                             columns=[str(unknownname)+'_vs_'+str(knownname)+'_peak_Scores Unnormalized'])
         compdf=compdf.assign(Peaks=score_sort(unknown_peaks,known_peaks,1, precision)[0][1][:])
     return compdf
+
 def score_table(unknown_peaks,known_peaks, precision,unknownname,knownname):
     "documentation"
```
Comment (erasmuss) [inline ramandecompy/peakidentify.py]: need doc string here, please keep to format laid out -- see parker's functions for good doc string examples. should include defining the input, and return and purpose

File: ramandecompy/peakidentify.py [hunk 2]
```diff
@@ -600,6 +597,7 @@ def score_sort(row_i, row_j, k, precision):
     sortedscores.sort()
 
     return sortedscores
+
 def process_score(unknown_peaks,known_peaks,k, precision, unknownname, knownname):
     "documentation"
```
Comment (erasmuss) [inline ramandecompy/peakidentify.py]: need doc string here

Comment (erasmuss) [reply ramandecompy/peakidentify.py]: @kernb2 please review comments and update :)

Comment (kernb2) [reply ramandecompy/peakidentify.py]: made doc string

File: ramandecompy/peakidentify.py (hunk 1 above)
Comment (kernb2) [reply ramandecompy/peakidentify.py]: made doc string

File: ramandecompy/tests/test_peakidentify.py [hunk 3]
```diff
@@ -702,8 +710,87 @@ def test_score_sort():
         is sorted from smallest to largest"""
         assert arrsortedscores[0][0][i] <= arrsortedscores[0][0][i+1], """Output
         values is sorted from smallest to largest"""
+        
 def test_process_score(unknown_peaks,known_peaks,k, precision, unknownname, knownname):
     "documentation"
```
Comment (erasmuss) [inline ramandecompy/tests/test_peakidentify.py]: need doc string

File: ramandecompy/tests/test_peakidentify.py [hunk 4]
```diff
@@ -702,8 +710,87 @@ def test_score_sort():
...
+    for _,peak in enumerate(list(calhdf5[key5])[:-3]):
+        H2O_peaks.append(list(calhdf5['{}/{}'.format(key5, peak)])[0][2])
+    precision = 10
+    unknown_name = 'Formic3.6wt%'
+    knownname = 'H2O'
+    # run good examples
+    process_score(unknown_peaks,known_peaks,k, precision,unknownname,knownname)
+    #run bad inputs
+    try:
+
```
Comment (erasmuss) [inline ramandecompy/tests/test_peakidentify.py]: extra line

File: ramandecompy/tests/test_peakidentify.py [hunk 5]
```diff
@@ -702,8 +710,87 @@ def test_score_sort():
...
+        process_score(unknown_peaks,known_peaks,k, precision,unknownname,3)
+
+    except TypeError:
+
+        print("An invalid maxscores from score_max was passed to the function, "
+              "and was handled correctly.")
+    # make assertions
+
```
Comment (erasmuss) [inline ramandecompy/tests/test_peakidentify.py]: lots of extra lines like 20 at least in this code block

File: ramandecompy/tests/test_peakidentify.py [hunk 6]
```diff
@@ -716,12 +803,89 @@ def test_process_score(unknown_peaks,known_peaks,k, precision, unknownname, know
     return
 def test_score_table(unknown_peaks,known_peaks, precision,unknownname,knownname):
     "documentation"
```
Comment (erasmuss) [inline ramandecompy/tests/test_peakidentify.py]: need doc string

File: ramandecompy/tests/test_peakidentify.py [hunk 7]
```diff
@@ -716,12 +803,89 @@ def test_process_score(unknown_peaks,known_peaks,k, precision, unknownname, know
...
+    try:
+
+        score_table(unknown_peaks,H2O_peaks,k, precision,unknownname,3)
+
+    except TypeError:
+
+        print("An invalid maxscores from score_max was passed to the function, "
+              "and was handled correctly.")
+    # make assertions
+
```
Comment (erasmuss) [inline ramandecompy/tests/test_peakidentify.py]: again, this code section has over 10 extra lines between lines, please delete

**Pull request merged**